### ISSUE TITLE
knex.transaction(function).finally(function) breaks in 0.7.x, but works in 0.5.x

### ISSUE THREAD

**Issue by blai** [tgriesser/knex#767]
It seems in 0.5.x, you can do this:

```js
knex.transaction(function(trx) {
    // handles transaction
}).finally(function() {
    //do something
})
```

But this breaks in 0.7.x (or may even 0.6.x), the [documentation](http://knexjs.org/#Upgrading) said otherwise.

**Comment by tgriesser** [comment]
Sorry about that - will get this fixed very soon

**Comment by tgriesser** [comment]
Should be fixed in 0.8
